src/main/java/org/broadinstitute/hellbender/utils/read/ReadUtils.java

```diff
@@ -74,6 +156,62 @@ public static String prettyPrintSequenceRecords ( SAMSequenceDictionary sequence
         RIGHT_TAIL
     }
 
+    public static final int SAM_READ_PAIRED_FLAG = 0x1;
```

Comment:
do these constants belong here? This seems like a very odd place for them? Doesnt htsjdk have a home for them?